test/core/bitswap.spec.js

```diff
@@ -16,10 +16,9 @@ const isNode = require('detect-node')
 const multihashing = require('multihashing-async')
 const CID = require('cids')
 
-const DaemonFactory = require('ipfsd-ctl')
-const df = DaemonFactory.create({ type: 'js' })
-
-const dfProc = DaemonFactory.create({ type: 'proc' })
+const IPFSFactory = require('ipfsd-ctl')
+const fDaemon = IPFSFactory.create({ type: 'js' })
```

Comment:
I find `fDaemon` a strange, hard to read, uncommon name. What about just `Deamon` or `jsDeamon`.

Reply:
It's a Factory. I could call it factoryDaemon if that makes more sense for you

Reply:
Does it matter that's a factory? If you really want to know, I think it's apparent from `IPFSFactor.create()` that it is a factory.
This don't find this super important, so if you don't find it worth the change, feel free to leave it as it currently is.